dev/run-dependencies.sh

```diff
@@ -13,7 +13,7 @@ echo "Dependencies are now running; press Ctrl+C to teardown"
 
 # From now on, if the user presses Ctrl+C we should teardown
 function cleanup() {
-    docker-compose -f $here/../scripts/docker-compose.yml --project-name montagu down
+    docker-compose --compatibility -f $here/../scripts/docker-compose.yml --project-name montagu down
```

Comment:
unrelated, but new docker version uses a different naming convention for containers brought up with docker compose. this flag retains the old style naming convention